content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -111,18 +111,10 @@ We obtained these results using the [Istio benchmarks](https://github.com/istio/
 Note: This testing was performed on the [CNCF Community Infrastructure Lab](https://github.com/cncf/cluster).
 Different hardware will give different values.
 
-<img width="90%" style="display: block; margin: auto;"
-    src="istio-1.24.0-fortio-90.png"
-    alt="P90 latency vs client connections"
-    caption="P90 latency vs client connections"
-/>
-<br><br>
-<img width="90%" style="display: block; margin: auto;"
-    src="istio-1.24.0-fortio-99.png"
-    alt="P99 latency vs client connections"
-    caption="P99 latency vs client connections"
-/>
-<br>
+{{< figure src="istio-1.24.0-fortio-90.png" caption="P90 latency vs client connections" width="90%" >}}
```

Comment:
We have a custom `{{< image >}}` shortcode.  We should use that, and then later on try and figure out if we need to maintain it or can just move to using the Hugo provided variant.

 ```suggestion
{{< image link="./istio-1.24.0-fortio-90.png" caption="P90 latency vs client connections" width="90%" >}}
```

(The `./` is probably not needed but many other examples use it)